tools/make/run.mk

```diff
@@ -13,7 +13,6 @@ run-server: ## Run the API Testing server
 run-server: build-ui run-backend
 run-backend:
 	go run . server --local-storage 'bin/*.yaml' --console-path ${ATEST_UI}/dist \
-		--mock-config bin/mock.yaml \
```

Comment:
why delete this? 🤔 
this doesn't seem to have anything to do with this PR

Reply:
`--mock-config` is conflict with `--local-storage`. You can try it.